sdk/include/reactos/layout.h

```diff
@@ -0,0 +1,232 @@
+/*
+ * PROJECT:     ReactOS headers
+ * LICENSE:     LGPL-2.0-or-later (https://spdx.org/licenses/LGPL-2.0-or-later)
+ * PURPOSE:     The layout engine of resizable dialog boxes / windows
```

Comment:
There is also `base/applications/rapps/include/rosui.h` which was supposed to be used in more places than only rapps, but it might not apply here.
Just something to keep in mind :)

Reply:
This layout.h file is also nice, in case one wants to do some layout calculation in a C program 😂